Detectors/Raw/include/DetectorsRaw/RDHUtils.h

```diff
@@ -138,7 +138,7 @@ struct RDHUtils {
   template <typename RDH>
   static void setMemorySize(RDH& rdh, uint16_t v)
   {
-    rdh.offsetToNext = v;
+    rdh.memorySize = v;
```

Comment:
isn't that version dependent?

Reply:
Perhaps you are right, for RDHV2 memorySize doesn't exist, from RDH4 onward it is correct. But then, RDHV2 doesn't have offsetToNext either.
Actually, do we still need to support < 4?